src/app/bindings.ts

```diff
@@ -0,0 +1,36 @@
+import {AppBinding, AppState, AppsState} from 'mattermost-redux/types/apps';
+
+class Bindings {
+    // getBindings returns bindings defined for all locations in the app
+    getBindings(): AppState {
+        const state: AppsState = [
+            this.postMenuBindings(),
+        ];
+        return state;
+    }
+
+    // postMenuBindings returns bindings for the post_menu location
+    postMenuBindings(): AppBinding {
```

Comment:
This can be in a flat function `getPostMenuBindings`, as `this` is never used in this method.

Reply:
Agree and flattening this file.